.github/workflows/docs.yml

```diff
@@ -0,0 +1,107 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  docs-detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      path-filter: ${{ steps.filter.outputs.path-filter }}
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'pull_request'
+      - uses: dorny/paths-filter@v2
+        if: github.event_name == 'pull_request'
+        id: filter
+        with:
+          filters: |
+            path-filter:
+              - './docs/**'
+
+  # TODO: Add generation of Solidity documentation
+
+  docs-html:
+    runs-on: ubuntu-latest
+    needs: [docs-detect-changes]
+    if: | # `always()` returns true, even if job should be cancelled
+      always()
```

Comment:
What's the exact purpose of `always()` here?

Reply:
It was copied from `keep-core`. We use it in `keep-core` to continue docs publication no matter of the result of previous step `docs-solidity`. We don't do solidity docs generation here yet so I removed this `always()`. 92f224b